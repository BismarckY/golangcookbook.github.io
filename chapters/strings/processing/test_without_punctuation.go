package main

import (
	"fmt"
	"strings"
)

func main() {
	s := "This, that, and the other."
	replacer := strings.NewReplacer(",", "", ".", "", ";", "")
	s = replacer.Replace(s)
	words := strings.Fields(s)
	for i, word := range words {
		fmt.Println(i, " => ", word)
	}
}
