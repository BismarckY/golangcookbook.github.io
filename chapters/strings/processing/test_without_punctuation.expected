0  =>  This
1  =>  that
2  =>  and
3  =>  the
4  =>  other
